mulighetsrommet-api/src/main/resources/db/migration/V51__notifications.sql

```diff
@@ -0,0 +1,24 @@
+create type notification_type as enum ('Notification', 'Task');
+
+create type notification_target as enum ('All', 'User');
+
+create table notification
+(
+    id          uuid primary key    not null,
+    type        notification_type   not null,
+    target      notification_target not null,
+    title       text                not null,
+    description text,
+    created_at  timestamp           not null
+);
+
+create index notification_target_created_at_idx on notification (target, created_at desc);
+
+create table user_notification
+(
+    notification_id uuid not null references notification (id) on delete cascade,
+    user_id         text not null,
+    seen_at         timestamp,
```

Comment:
Mulig vi skulle kalt dette for `read` i stedet for `seen` på tvers av tjenestelaget..

Reply:
Enig i at `read` leses bedre, imho